app/controllers/idv/in_person/ready_to_verify_controller.rb

```diff
@@ -17,8 +17,9 @@ class ReadyToVerifyController < ApplicationController
       before_action :confirm_in_person_session
 
       def show
+        @is_eipp = sp_session.key?('vtr') && sp_session['vtr'].first&.include?('Pe')
```

Comment:
Is there any way we can use the parsed version of the VOTs?

Reply:
Yes- I am actually planning on refactoring this and using the service Eileen set up in [PR# 10750](https://github.com/18F/identity-idp/pull/10750/files) once she gets it merged in so we all have the same logic.

Reply:
based on the fixes there, this can be:
```suggestion
        @is_eipp = resolved_authn_context_result.enhanced_ipp?
```

Reply:
Yes! I just figured that after looking at Eileen's PR.  I made it more complicated and deleted all my code. Thanks for the suggestion too! 🥳  Resolved with commit 5b996577b2c85908d21510e51fdb582d74c1b4aa